### DISCUSSION TITLE
accuracy_analysis_from_labels_table not working

### DISCUSSION THREAD

**Discussion by bennyhere** [moj-analytical-services/splink#2421] [Q&A]
I was able to run linker.truth_space_table_from_labels_table(labels_table)

but

#below funcion is not working!!!
linker.accuracy_analysis_from_labels_table(
    labels_table, output_type="threshold_selection", add_metrics=["f1"]
)

AttributeError: 'SparkLinker' object has no attribute 'accuracy_analysis_from_labels_table'

**Comment by RobinL** [comment]
I think you're trying to run splink 3 code but you have splink 4 installed.  See 
https://moj-analytical-services.github.io/splink/api_docs/evaluation.html

the functions are now at `linker.evaluation.`...
